build.gradle

```
@@ -600,15 +600,7 @@ repositories {
         }
         maven {
             name = "ic2"
-            url = "https://maven.ic2.player.to/"
-            metadataSources {
-                mavenPom()
-                artifact()
-            }
-        }
-        maven {
-            name = "ic2-mirror"
-            url = "https://maven2.ic2.player.to/"
+            url = getURL("https://maven.ic2.player.to/", "https://maven2.ic2.player.to/")
```

Let's also add a filter to prevent this repo being offline from messing with other dependency resolution
```suggestion
            url = getURL("https://maven.ic2.player.to/", "https://maven2.ic2.player.to/")
            filter {
                includeGroup "net.industrial-craft"
            }
```